oap-stdlib/src/main/java/oap/reflect/SystemTagsService.java

```diff
@@ -0,0 +1,92 @@
+package oap.reflect;
+
+import lombok.extern.slf4j.Slf4j;
+import oap.util.Strings;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class is able to collect system labels (which are any objects, like enums, strings, etc).
+ * In order to process class it should be annotated with @SystemTagSupport.
+ * Any field/method annotated with @SystemTag is processed. If system label gives a name, like
+ *
+ * @SystemTag( "Private property" ), then it will have such a category name, otherwise its name
+ * will be assembled from field/method name. The result of processing is a map, where keys are
+ * category names, and values are actual values of fields/methods.
+ */
+@Slf4j
+public class SystemTagsService {
```

Comment:
What is it for?
`org.apache.commons.lang3.reflect.FieldUtils.getFieldsListWithAnnotation(clazz, MyAnnotation)`